extensions/resteasy-reactive/quarkus-resteasy-reactive-kotlin/runtime/pom.xml

```diff
@@ -25,18 +25,25 @@
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-stdlib-jdk8</artifactId>
-            <optional>true</optional>
         </dependency>
         <dependency>
             <groupId>org.jetbrains.kotlinx</groupId>
             <artifactId>kotlinx-coroutines-core-jvm</artifactId>
-            <optional>true</optional>
         </dependency>
     </dependencies>
 
     <build>
         <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
         <plugins>
+            <plugin>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
+                <configuration>
+                    <dependencyCondition>
+                        <artifact>io.quarkus:quarkus-kotlin</artifact>
+                    </dependencyCondition>
```

Comment:
This configures a condition for the extension to be enabled. It can actually be any artifact, not necessarily an extension, or a set of artifacts.